### ISSUE TITLE
File .csl not found in resource path

### ISSUE THREAD

**Issue by nmueller18** [ISAAKiel/mortAAR#61]
In the latest CRAN-tests, there surfaced an [error](https://cran.r-project.org/web/checks/check_results_mortAAR.html) in the processing of the references in the vignettes via pandoc. It has to be fixed by 2022/05/06:
```
Check: re-building of vignette outputs 
Result: ERROR 
    Error(s) in re-building vignettes:
    --- re-building 'mortAAR_vignette-1.Rmd' using rmarkdown
    File .csl not found in resource path
    Error: processing vignette 'mortAAR_vignette-1.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 99
    --- failed re-building 'mortAAR_vignette-1.Rmd'
Error: Vignette re-building failed.
    Execution halted 
Flavor: [r-release-windows-x86_64]
```
A Google-search did not bring up obvious solutions. The error probably implies that pandoc cannot access the csl-file linked in the vignettes. The error only occurs on the test Windows system, but also on my Mac the command `RCurl::url.exists('https://www.zotero.org/styles/offa')` used in the vignettes returns `FALSE`, which of course leaves the text string "`r library(RCurl); ifelse(url.exists('https://www.zotero.org/styles/offa'), 'https://www.zotero.org/styles/offa', '')`" empty. The corresponding command of the package `httr` `httr::GET('https://www.zotero.org/styles/offa')` strangely returns `Error in curl::curl_fetch_memory(url, handle = handle) : SSL certificate problem: certificate has expired`. However, a test of the address yielded a valid certificate. Increasing the `connecttimeout ` setting of RCurl did not help, either.
I am puzzled. One possible solution seems to bundle a copy of the offa-csl-file with the package but this would defy the whole idea of csl-styles somehow. Any other ideas?

**Comment by MartinHinz** [comment]
I am puzzled, too, because at my (linux) system, it looks like this:
![image](https://user-images.githubusercontent.com/3285169/164974915-86c79dca-343a-47f4-8af5-4bbff91e4c7d.png)

I suggest removing the `csl: "r library(RCurl); ifelse(url.exists('https://www.zotero.org/styles/journal-of-archaeological-science'), 'https://www.zotero.org/styles/journal-of-archaeological-science', '')`"` alltogether, to avoid that error. What do you think?

**Comment by nmueller18** [comment]
But in this case the csl-file has to be bundled with the package, or am I wrong?

**Comment by MartinHinz** [comment]
We do not need the csl, or only, if we want to display that style, I suppose.

**Comment by nmueller18** [comment]
The style is used by pandoc for knitting the vignettes, as far as I know.

**Comment by MartinHinz** [comment]
That might be true, but we could go without. Than, the references would look like this:
![image](https://user-images.githubusercontent.com/3285169/164976517-b8c9c343-8c23-478d-9e4b-2cb98fb16362.png)

Compare to the current one at the vignette on CRAN:
![image](https://user-images.githubusercontent.com/3285169/164976552-1aa34da8-c7ad-4272-8d69-c780c856a0d3.png)

https://cran.r-project.org/web/packages/mortAAR/vignettes/mortAAR_vignette-1.html

I do not think this is a dramatic loss.

**Comment by nmueller18** [comment]
Ah, in this case, the line should go, as this is the easiest solution. I will prepare a new version.

**Comment by nmueller18** [comment]
Fixed in version 1.1.2.
